src/ui/popupmenu/popupmenu.rs

```diff
@@ -256,6 +258,7 @@ impl Popupmenu {
 
         Popupmenu {
             items: LazyLoader::new(list.clone(), css_provider.clone()),
+            menu_on_all_items: false,
```

Comment:
Will set this to default to `true`, as per @vhakulinen in https://github.com/vhakulinen/gnvim/issues/41#issuecomment-530921001.